community/kernel/src/main/java/org/neo4j/graphdb/Result.java

```diff
@@ -178,4 +180,14 @@
 
     /** Removing rows from the result is not supported. */
     void remove();
+
+    /**
+     * Provides notifications about the execution of the query producing this result.
```

Comment:
not sure about `execution`, we don't really execute the query since notifications are enabled only for `explain`